### DISCUSSION TITLE
"The given assembly name was invalid." After creating new Module

### DISCUSSION THREAD

**Discussion by Justincale** [oqtane/oqtane.framework#3215] [Q&A]
Hi guys, i recently started looking at Oqtane but have run into an issue. I am running the latest version from github on .Net 7. After creating a new module from within oqtane i opened the newly created module and ran a build. Now when i try to start an instance of Oqtane i recieve the following exception from within LoadModuleDefinitionsFromAssembly: "The given assembly name was invalid."

The Module created has the following:

ServerManagerType = "test.Module.mymod, test.Module.mymod.Client.Oqtane.Manager.mymodManager, test.Module.mymod, test.Module.mymod.Client.Oqtane.Server.Oqtane".

Any help would be grand.

Thanks

**Comment by sbwalker** [comment]
@Justincale There was a change in 4.0.3 which caused this issue. Change the ServerManagerType value to a valid fully qualified type and recompile the module:

ServerManagerType = "test.Module.mymod.Manager.mymodManager, test.Module.mymod.Server.Oqtane"

This will be fixed in 4.0.4

**Reply by Justincale** [reply]
Will do, thanks

**Comment by sbwalker** [comment]
@Justincale fixed in #3217 - thank you for reporting
